### ISSUE TITLE
Affichage vidéo template espèce

### ISSUE THREAD

**Issue by MNourrisson** [PnX-SI/GeoNature-atlas#241]
Bonjour,
une petite erreur s'est glissée sur le template espèce :
https://github.com/PnX-SI/GeoNature-atlas/blob/fa0d2f60eff91655d32123f57a534263d931f126/templates/ficheEspece.html#L528

Je l'ai remplacée par 
`<source` src="{{videoAudio.video[0].path}}" `type="video/mp4">`

Bonne journée

**Comment by Splendens** [comment]
Effectivement, cette ligne créée une erreur ` jinja2.exceptions.UndefinedError: 'video' is undefined
`
Merci beaucoup pour la correction !
